templates/init_runner_script.sh

```diff
@@ -85,6 +85,11 @@ elif [ "${LINUX_OS_VERSION}" = "8" ]
 then
 sudo dnf install -y podman-docker
 DOCKER_SERVICE_START="no"
+elif [ "${LINUX_OS_VERSION}" = "9" ]
+then
```

Comment:
I'm glad to see this worked, but this troubles me. It's a very different behavior from what was done with rhel8. 

rhel does not provide docker since version 8 and distrubutes podman instead, so the choice has been made to install podman, along with `podman-docker` which provides an alias to podman as the commands practically the same and hope for the best. But most importantly, someone that chose to use rhel should not be using docker.

You are now reverting this choice in rhel 9 and installing docker. IMHO, we should have installed podman-docker as well, with something like:
`[ "${LINUX_OS_VERSION}" = "8" || "${LINUX_OS_VERSION}" = "9" ]` on line 84